plugins/GSdx/Renderers/DX11/GSDevice11.cpp

```diff
@@ -168,8 +166,15 @@ bool GSDevice11::Create(const std::shared_ptr<GSWnd> &wnd)
 				hr = enum_adapter->GetDesc1(&desc);
 				if (S_OK == hr && (GSAdapter(desc) == adapter_id || adapter_id == "default"))
 				{
+					// D3D GPU vendor check.
 					if (desc.VendorId == 0x10DE)
-						nvidia_vendor = true;
+						m_d3d_gpu_vendor = D3D11_VENDOR_NVIDIA;
```

Comment:
Don't use `D3D11_` namespace. It is reserved by the api.
I don't think we need the `d3d_` prefix either.